src/containers/BottomCarousel.tsx

```diff
@@ -0,0 +1,8 @@
+import FiveFilmsCarousel from '../components/FiveFilmsCarousel';

```

Comment:
Why do we need this component? I think it's not doing anything, so can be removed. Instead, you can import FiveFilmsCarousel directly to dashboard.